src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -102,11 +123,14 @@ export const HookValidatedInput = ({
   type = 'text',
   inputType,
   warning = undefined,
+  isEmpty,
```

Comment:
I don't think this should be needed.

Reply:
why not?
without it, if there is no value we see the sucess status or error message, if user decide to delete the value 
<img width="658" alt="Screenshot 2025-02-09 at 14 53 00" src="https://github.com/user-attachments/assets/d53cbe7d-9f1d-44bf-a9c1-ca511c89fc8f" />

Reply:
The "emptiness" of the input is already handled by the `isPristine` state. I've tried to remove the `isEmpty` and the behaviour of the field didn't change in any way.